email/parser.go

```diff
@@ -93,6 +104,23 @@ func parseMessageWithHeader(headers Header, bodyReader io.Reader) (*Message, err
 	}, nil
 }
 
+// removeDuplicate removes duplicate from bogus content type preventing parsing of email
+func removeDuplicate(s string) string {
+	m := make(map[string]bool)
+	parts := strings.Split(s, ";")
```

Comment:
I need to re-lookup content types, but isn't the very first entry a special case?
Golang map keys are random, so that `image/png` entry might get put at the back of the list.